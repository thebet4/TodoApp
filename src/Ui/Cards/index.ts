export { default as HotelCard } from './HotelCard';
export { default as TeamCard } from './TeamCard';
