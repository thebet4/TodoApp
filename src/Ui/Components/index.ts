export { default as Header } from './Header'
export { default as TabBarItem } from './TabBarItem'
