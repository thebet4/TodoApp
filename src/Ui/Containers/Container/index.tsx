import React, { ReactChild, ReactChildren } from 'react';
import { StatusBar as RNStatusBar, Platform } from 'react-native';
import { StatusBar } from 'expo-status-bar';
import {
  SafeAreaView,
  KeyboardAvoidingView,
} from './style';

interface IContainerProps {
  children?: ReactChild | ReactChildren;
}

const Container = ({ children }: IContainerProps) => {
  return (
    <SafeAreaView style={{
      paddingTop: RNStatusBar.currentHeight,
    }}
    >
      <StatusBar style="dark" />
      <KeyboardAvoidingView
        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
      >
        {children}
      </KeyboardAvoidingView>
    </SafeAreaView>
  );
};

export default Container;
