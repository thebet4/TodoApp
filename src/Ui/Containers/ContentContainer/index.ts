import styled from 'styled-components/native';

export const ContentContainer = styled.View`
  flex:1%;
  padding: 24px;
`;
