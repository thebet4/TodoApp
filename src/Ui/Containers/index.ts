export { default as Container } from './Container';
export { ContentContainer } from './ContentContainer';
