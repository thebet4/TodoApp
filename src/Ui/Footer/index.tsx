import React from 'react';
import {
  ImageBackground, View, TouchableOpacity, Image,
} from 'react-native';
import { Colors } from '../Theme';
import Caption from '../Typography/Caption';
import Title40 from '../Typography/Title40';

const Footer = () => {
  return (

    <ImageBackground
      source={require('./Assets/HomeFooterBg.png')}
      style={{ paddingBottom: 64 }}
    >
      <View style={{
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'center',
        marginTop: 92,
      }}
      >
        <View style={{
          width: '30%',
          backgroundColor: Colors.Base.Light,
          height: 1,
        }}
        />
        <Title40 style={{
          color: Colors.Base.Light,
          marginHorizontal: 12,
        }}
        >
          Partners
        </Title40>
        <View style={{
          width: '30%',
          backgroundColor: Colors.Base.Light,
          height: 1,
        }}
        />
      </View>

      <View style={{
        alignItems: 'center',
        justifyContent: 'space-around',
      }}
      >
        <Image source={require('./Assets/partner1.png')} resizeMode="center" />
        <Image source={require('./Assets/partner2.png')} resizeMode="center" />
        <Image source={require('./Assets/partner4.png')} resizeMode="center" />
        <Image source={require('./Assets/partner5.png')} resizeMode="center" />
        <Image source={require('./Assets/partner6.png')} resizeMode="center" />
        <Image source={require('./Assets/partner7.png')} resizeMode="center" />

      </View>

      <View style={{ width: '70%', alignSelf: 'center' }}>

        <TouchableOpacity style={{
          height: 44,
          alignItems: 'center',
          justifyContent: 'center',
          borderWidth: 2,
          borderColor: Colors.Base.Light,
          borderRadius: 51,
          marginBottom: 17,
        }}
        >
          <Caption style={{ color: Colors.Base.Light, fontSize: 18 }}>
            Acessar
          </Caption>
        </TouchableOpacity>
        <TouchableOpacity style={{
          height: 44,
          alignItems: 'center',
          justifyContent: 'center',
          borderRadius: 51,
          backgroundColor: Colors.Base.Light,
        }}
        >
          <Caption style={{ color: Colors.Primary.Normal, fontSize: 18 }}>
            Abrir Conta
          </Caption>
        </TouchableOpacity>
      </View>
    </ImageBackground>

  );
};

export default Footer;
