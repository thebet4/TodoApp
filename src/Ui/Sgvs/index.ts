export { default as CalendarIcon } from './CalendarIcon'
export { default as ListIcon } from './ListIcon'
export { default as SunIcon } from './SunIcon'
export { default as TaskIcon } from './TaskIcon'
export { default as SearchIcon } from './SearchIcon'
export { default as FilterIcon } from './FilterIcon'
export { default as AddIcon } from './AddIcon'
export { default as PersonalIcon } from './PersonalIcon'
export { default as PriorityIcon } from './PriorityIcon'
export { default as ClockIcon } from './ClockIcon'
export { default as CommentIcon } from './CommentIcon'
export { default as RepeatIcon } from './RepeatIcon'
