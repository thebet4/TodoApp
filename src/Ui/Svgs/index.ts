export { default as LogoIcon } from './LogoIcon';
export { default as OpenedEyeIcon } from './OpenedEyeIcon';
export { default as CalendarIcon } from './CalendarIcon';
export { default as SearchIcon } from './SearchIcon';
export { default as NFTIcon } from './NFTIcon';
export { default as HotelIcon } from './HotelIcon';
export { default as BlockChainIcon } from './BlockChainIcon';
export { default as LogoWithTextIcon } from './LogoWithTextIcon';
