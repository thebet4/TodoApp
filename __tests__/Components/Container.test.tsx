import renderer from 'react-test-renderer';
import React from 'react';

import { Container } from '../../src/Ui/Containers';

describe('<Container />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Container />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
