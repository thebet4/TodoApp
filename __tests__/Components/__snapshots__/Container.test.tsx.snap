// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
      Object {
        "paddingTop": null,
      },
    ]
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Array [
          Object {
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
          },
        ],
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  />
</RCTSafeAreaView>
`;
